worker/src/RTC/TransportCongestionControlServer.cpp

```diff
@@ -262,21 +297,35 @@ namespace RTC
 
 		// Increment packet count.
 		this->transportCcFeedbackPacket->SetFeedbackPacketCount(++this->transportCcFeedbackPacketCount);
+		this->transportCcFeedbackStartSeqNum = latestWideSeqNumber + 1;
+	}
 
-		// Pass the latest packet info (if any) as pre base for the new feedback packet.
-		if (latestTimestamp > 0u)
+	inline void TransportCongestionControlServer::MayDropOldPacketArrivalTimes(
+	  uint16_t seqNum, uint64_t nowMs)
+	{
+		MS_TRACE();
+
+		// Ignore nowMs value if it's smaller than 500 in order to avoid negative values
+		// (Should never happen) and return early if the condition is met.
```

Comment:
```suggestion
		// Ignore nowMs value if it's smaller than PacketArrivalTimestampWindow in
		// order to avoid negative values (should never happen) and return early if
		// the condition is met.
```